pypesto/objective/amici_util.py

```diff
@@ -237,11 +241,38 @@ def get_error_output(
                  for data in edatas)
     n_res = nt * amici_model.nytrue
 
-    return {
-        FVAL: np.inf,
-        GRAD: np.nan * np.ones(dim),
-        HESS: np.nan * np.ones([dim, dim]),
-        RES:  np.nan * np.ones(n_res),
-        SRES: np.nan * np.ones([n_res, dim]),
+    nllh = np.inf
+    snllh = None
+    s2nllh = None
+    if mode == MODE_FUN and sensi_order > 0:
+        snllh = np.nan * np.ones(dim)
+        s2nllh = np.nan * np.ones([dim, dim])
```

Comment:
I think for compliance with the non-error output mode, one should check for sensi_order > 1?

Reply:
I copy pasted the code from amici-calculator, might be that they are initialized but not modified. Will look into in more detail.